src/components/MoviePosterCard.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <div>
```

Comment:
The height issue comes about since this div is not 100% in height. You can either add 'h-full' to this div or just remove it altogether and have its direct child be the root element of this component